### PR TITLE
chore: release v4.6.0

### PR DESCRIPTION
**Pull request by Haroenv**
This pull request prepares the following release:
| Repository | Branch | Update | Change |
|---|---|---|---|
| [algoliasearch-client-javascript](https://github.com/algolia/algoliasearch-client-javascript) | releases/v4.6.0 | minor | [`4.5.1` → `4.6.0`](https://github.com/algolia/algoliasearch-client-javascript/compare/4.5.1...releases/v4.6.0) |

### Release Summary
This is going to be published with the following command:

| Dir | Command |
|---|---|
| packages/algoliasearch | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/cache-browser-local-storage | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/cache-common | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/cache-in-memory | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/client-account | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/client-analytics | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/client-common | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/client-recommendation | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/client-search | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/logger-common | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/logger-console | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/requester-browser-xhr | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/requester-common | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/requester-node-http | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/transporter | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |

### Merging Instructions
When merging this pull request, you need to **Squash and merge** and make sure that the title starts with `chore: release v4.6.0`.
<details>
<summary>See details</summary>

After that, a commit `chore: release v4.6.0` will be added and you or your CI can run `shipjs trigger` to trigger the release based on the commit.
Squash and merge
</details>

---

_This pull request is automatically generated by [Ship.js](https://github.com/algolia/shipjs)_.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## [v4.6.0](https://github.com/algolia/algoliasearch-client-javascript/compare/4.5.1...4.6.0)
+
+* chore: migrate release process to Ship.js (#1210) ([f82bd48](https://github.com/algolia/algoliasearch-client-javascript/commit/f82bd48)), closes [#1210](https://github.com/algolia/algoliasearch-client-javascript/issues/1210)
+* chore: update bundlesize ([c3f5d72](https://github.com/algolia/algoliasearch-client-javascript/commit/c3f5d72))
+* chore(TypeScript): avoid ts-ignore (#1190) ([5207d68](https://github.com/algolia/algoliasearch-client-javascript/commit/5207d68)), closes [#1190](https://github.com/algolia/algoliasearch-client-javascript/issues/1190)
```
Comment (Haroenv) [inline CHANGELOG.md:5]:
```suggestion
```

**Review by bodinsamuel — APPROVED** [review]
🙏🏻

**Review by eunjae-lee — APPROVED** [review]
looks good to me

**Review by eunjae-lee — CHANGES_REQUESTED** [review]
The tests are failing.

File: yarn.lock [hunk 2]
```diff
@@ -2,6 +2,110 @@
 # yarn lockfile v1
 
 
+"@algolia/cache-browser-local-storage@4.5.1":
```
Comment (Haroenv) [inline yarn.lock:5]: these seem to be added by shipjs not updating all versions (of dependencies), let's see what's different than in the script before

it's causing the test:build to fail 👍

**Pull request merged**